### DISCUSSION TITLE
Vailant: Aktualisierung Werte antriggern

### DISCUSSION THREAD

**Discussion by m-ask** [john30/ebusd#723] [Q&A]
Hallo!
Mir ist folgendes aufgefallen. Bei z.b. der Wärmemenge wird der Wert immer zu Mitternacht aktualisiert. Wenn ich mit der Bedinungseinheit den Wert anschaue wird hier eine sofortige Akualisierung angestossen. Kann man diese Aktualisierung auch über ebusctl anstossen.
Danke 
m-ask

**Comment by BenKrae** [comment]
Du kannst die Werte über folgenden Befehl direkt aktualisieren:
ebusctl read -f [Parameter]
"-f" steht für forced und erzwingt mit Absenden des Befehls die Aktualisierung des Parameters
z.B.
ebusctl read -f Hc1HeatCurve

**Reply by m-ask** [reply]
Danke für die Antwort.
Kann das sein, dass das nicht bei jeder Größe funktioniert?
ebusctl -f YieldSum liefert keinen aktuellen Wert
andere Werte zeigen eine Veränderung

**Reply by JeeCee1** [reply]
ebusctl **read** -f YieldSum

**Comment by minitrue-nl** [comment]
Did you manage to make this work @m-ask ?

I see especially on Energy Counters (PrEnergySum on 700 or ConsumptionTotal on HMU) that data is only updated at midnight. I'm requesting data per MQTT wit /get, so I think that should force an actual reading no?

**Reply by minitrue-nl** [reply]
Also double checked with command `ebusctl read -f PrEnergySum` but output number is the same all day. Only after 0.00h it will update with the new number of the day before.

**Reply by minitrue-nl** [reply]
@john30 Maybe you can enlighten me a bit here. Is there a way to force a refresh reading from the vrc700? When I look manually at the device, the total PrEnergySum is updated, but this is not the case when I call on the value via TCP or MQTT. Am I missing another option to refresh/reload?
